.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+      - name: Run tests
+        run: pytest -q 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)